labels.json

```diff
@@ -1,135 +1,316 @@
-    "color": "eb6420",
+    "name": "support",
+    "description": "Issue with a support-like question",
+    "color": "ebefc9",
     "aliases": [
-      "backwards-incompatible",
-      "breaking"
+      "support",
+      "question"
     ]
   },
   {
-    "name": "beginner-friendly",
-    "color": "d4c5f9",
+    "name": "new functionality request",
+    "description": "Issue with a request to add functionality",
+    "color": "006b75",
     "aliases": [
-      "beginner",
-      "beginner friendly"
+      "new functionality request",
+      "enhancement",
+      "feature"
+    ]
+  },
+  {
+    "name": "bug report",
+    "description": "Issue raised reporting a potential bug",
+    "color": "f7c6c7",
+    "aliases": [
+      "bug report",
+      "bug"
     ]
   },
   {
-    "name": "bug",
+    "name": "bug fix",
+    "description": "Pull Request raised fixing a potential bug",
     "color": "0e8a16",
     "aliases": [
-      "bug",
+      "bug fix",
       "bugfix"
     ]
   },
   {
     "name": "docs",
-    "color": "006b75",
+    "description": "Any Issue/Pull Request that related to documentation",
+    "color": "fef2c0",
     "aliases": [
       "docs",
       "documentation"
     ]
   },
   {
-    "name": "duplicate",
-    "color": "cccccc",
+    "name": "technical-debt",
+    "description": "Any Issue that raises changes that should be done, and any Pull Request that does them",
+    "color": "c7def8",
     "aliases": [
-      "duplicate"
+      "technical-debt",
+      "debt"
+    ]
+  },
+  {
+    "name": "status: needs more information",
+    "description": "An Issue reporting a potential bug that needs more detail in order to help reproduce it",
+    "color": "c6a49a",
+    "aliases": [
+      "status: needs more information",
+      "needs more information",
+      "needs-more-info"
     ]
   },
   {
-    "name": "enhancement",
+    "name": "resolution: can't reproduce",
+    "description": "An Issue reporting a bug that cannot be reproduced",
     "color": "0052cc",
     "aliases": [
-      "enhancement",
-      "feature"
+      "resolution: can't reproduce",
+      "non-reproducible"
     ]
   },
   {
-    "name": "modulesync",
-    "color": "fbca04",
+    "name": "resolution: duplicate",
+    "description": "An Issue raised that has already been reported",
+    "color": "cccccc",
     "aliases": [
-      "msync"
+      "resolution: duplicate",
+      "duplicate"
     ]
   },
   {
-    "name": "needs-docs",
-    "color": "006b75",
+    "name": "resolution: fixed",
+    "description": "An Issue that has been fixed by a Pull Request",
+    "color": "d4c5f9",
     "aliases": [
-      "needs-docs"
+      "resolution: fixed"
     ]
   },
   {
-    "name": "needs-feedback",
-    "color": "5319e7",
+    "name": "resolution: invalid",
+    "description": "An Issue that is determined to not be related to this project",
+    "color": "e6e6e6",
     "aliases": [
-      "feedback",
-      "needs-feedback"
+      "resolution: invalid",
```

Comment:
Will not help to have a label. Closing with a clear explanation of WHY invalid is more appropriate.